api/events/handlers.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
```

Comment:
I personally like more call this file as a controller, because is the same, but with other name :smile: